### PR TITLE
Fix sheets resize race condition

### PR DESCRIPTION
**Pull request by bgraves-lo**
* We need to wait for retriableResize to complete before attempting to paste our new data.

### CODE REVIEW

File: lib/actions/google/analytics/lib/csv_header_transform_stream.d.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /// <reference types="node" />
 /// <reference types="node" />
+/// <reference types="node" />
```
Comment (bgraves-lo) [inline lib/actions/google/analytics/lib/csv_header_transform_stream.d.ts:3]: Seems like cruft being added by `yarn build`?

Comment (phillipperalez) [reply lib/actions/google/analytics/lib/csv_header_transform_stream.d.ts:3]: should be fine

**Review by drstrangelooker — DISMISSED** [review]
LGTM

**Review by phillipperalez — APPROVED** [review]
LGTM, letting Jim see this too

**Pull request merged**